src/client/social.radiatus.js

```diff
@@ -16,25 +16,21 @@
  * @param {WebSocket} webSocket Alternative webSocket implementation for tests
  **/
 var DEBUGLOGGING = false;
+var WS_URL = 'ws://localhost:8082/route/';
+//var WS_URL = 'wss://data.radiatus.io/route/';
```

Comment:
It looks like hardcoding it here changes the logic a bit, i.e. makes it so WS_URL is always localhost rather than being localhost when DEBUG is true and radiatus.io otherwise. If that's intentional I'm okay with that, but it may still be nice to have some sort of flag to select or pass in a custom WS host.